app/controllers/feedback_controller.rb

```diff
@@ -12,21 +13,15 @@ def new
   def create
     @feedback = Feedback.new(merged_feedback_params)
     if @feedback.save
-      submit_feedback_google_event if @feedback.is?('feedback')
       redirect_to after_create_url, notice: 'Feedback submitted'
     else
+      flash[:error] = @feedback.error
```

Comment:
Feedback and Bug report both share the same controller.

The bug report view uses `govuk_error_summary` to display errors, therefore this flash would also show on the bug report view, you may need to an a conditional check for feedback type.

![Screenshot 2021-09-30 at 14 57 46](https://user-images.githubusercontent.com/8057224/135472478-e9fca353-95c8-4035-ad05-75d2c6c51422.png)

Reply:
@jrmhaig

Reply:
is this resolved?

Reply:
Yes: 404f039a71445f247d2a45e86396f21f2c2c5305